Shared/en.lproj/Localizable.strings

```diff
@@ -116,7 +116,7 @@
 "Stop Sharing" = "Stop Sharing";
 
 /* No comment provided by engineer. */
-"Stop sharing after files have been sent (uncheck to allow downloading individual files)" = "Stop sharing after files have been sent (uncheck to allow downloading individual files)";
+"Stop sharing after files have been sent (uncheck to allow downloading individual files)" = "Stop sharing after files are sent. (Otherwise allows downloading individual files.)";
```

Comment:
```suggestion
"Stop sharing after files have been sent (uncheck to allow downloading individual files)" = "Stop sharing after sending everything.)";
```
Is the notion that this packs everything into one file missed?
~… after sending everything as a ZIP file?